### PR TITLE
feat: add toggle callback to Ellipsis component

### PR DESCRIPTION
**Pull request by damonyoungcc**
close https://github.com/ant-design/ant-design-mobile/issues/6570

### CODE REVIEW

**Comment by damonyoungcc** [comment]
close https://github.com/ant-design/ant-design-mobile/issues/6570

**Comment by damonyoungcc** [comment]
这里我只是先发出来讨论一下，感觉用toggle这个名字不太好
1. 或者加上受控的方案，用onChange
2. 或者参考[arco](https://arco.design/mobile/react/arco-design/pc/#/components/ellipsis)，增加`onEllipsisNodeClick` 和 `onCollapseNodeClick `

File: src/components/ellipsis/ellipsis.tsx [hunk 1]
```diff
@@ -21,6 +21,10 @@ export type EllipsisProps = {
   stopPropagationForActionButtons?: PropagationEvent[]
   onContentClick?: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void
   defaultExpanded?: boolean
+  toggle?: (
```
Comment (zombieJ) [inline src/components/ellipsis/ellipsis.tsx:24]: Could we use `onEllipsis` instead. It's same as the antd API: 
https://ant.design/components/typography-cn#api

Comment (damonyoungcc) [reply src/components/ellipsis/ellipsis.tsx:24]: Indeed, the design of onEllipsis in antd is more reasonable. The correct logic is to trigger onEllipsis based on whether the text is ellipsized or not. I'll continue working on it.

**Comment by zombieJ** [comment]
Why close?

**Pull request closed without merging**